Formula/serverless.rb

```diff
@@ -19,6 +19,10 @@ class Serverless < Formula
   def install
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
     bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    # Delete pre-compiled Linux CPython library that is included in dependency package
+    plugin = libexec/"lib/node_modules/serverless/node_modules/@serverless/dashboard-plugin"
+    rm "#{plugin}/sdk-py/serverless_sdk/vendor/wrapt/_wrappers.cpython-37m-x86_64-linux-gnu.so"
```

Comment:
I'm hoping this will make sure it stays gone on updates/
```suggestion
    rm "#{plugin}/sdk-py/serverless_sdk/vendor/wrapt/_wrappers.cpython-*-*-linux-gnu.so"
```

Reply:
Alternatively
```suggestion
    (plugin/"sdk-py/serverless_sdk/vendor/wrapt/_wrappers.cpython-37m-x86_64-linux-gnu.so").unlink
```
so we know whenever we're trying to `rm` something that's not there.

Reply:
In this case, the `rm` path is a single file, so it will cause build to fail if it no longer exists, e.g. I changed name from `.so` to `.soo` and got:
```
==> npm install -ddd --global --build-from-source --cache=/Users/cho-m/Library/Caches/Homebrew/npm_cache --prefix=/usr/l
Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory @ apply2files - /usr/local/Cellar/serverless/2.55.0/libexec/lib/node_modules/serverless/node_modules/@serverless/dashboard-plugin/sdk-py/serverless_sdk/vendor/wrapt/_wrappers.cpython-37m-x86_64-linux-gnu.soo
```

However, I think if we glob, `rm` will pass even if 0 files match.

I was debating between a catchall glob and a single file. For general maintenance, a glob is probably easier. For avoiding stale installation steps, using something that will cause CI to fail would be better.

Reply:
We can do something in between: use a glob, but `raise` it our glob doesn't pick up exactly one file.

Reply:
Added some logic for this, but not sure about preferred style here.